h2o-py/h2o/frame.py

```diff
@@ -445,28 +445,29 @@ def type(self, col):
 
 
     def _import_parse(self, path, pattern, destination_frame, header, separator, column_names, column_types, na_strings,
-                      skipped_columns=None, custom_non_data_line_markers=None, partition_by=None, quotechar=None):
+                      skipped_columns=None, custom_non_data_line_markers=None, partition_by=None, quotechar=None, escape_char=None):
```

Comment:
Should probably be consistent with naming of `quotechar`, Pandas also uses spelling `escapechar`